### ISSUE TITLE
[Bug]: Port 5006 is forced within docker

### ISSUE THREAD

**Issue by hrv231** [actualbudget/actual-server#313]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

Hello,

If I give my Actual docker its own IP address,  the docker forces me to use port 5006, instead of 443 or 80.

Example:
```
services:
  actual_server:
    image: docker.io/actualbudget/actual-server:latest
    ports:
      - '443:5006'
    environment:
      - ACTUAL_HTTPS_KEY=/data/selfhost.key
      - ACTUAL_HTTPS_CERT=/data/selfhost.crt
    volumes:
      - ./actual-data:/data
    restart: unless-stopped
    networks:
      my-network:
        ipv4_address: 10.10.10.10

networks:
  my-network:
    external: true
    name: my-network
```

### What error did you receive?

No error, but I think we should edit the Dockerbuild to not force port 5006, or allow an environment variable for when you are not using "bridge" network within docker.

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Firefox

### Operating System

Linux

**Comment by intiplink** [comment]
Can't test this as my docker instalation in armv6 is busted, but from your docker compose file, you're missing port environment variables https://actualbudget.org/docs/config/#port

**Comment by psybers** [comment]
I don't think this is a problem.  You need to set the `ACTUAL_PORT` environment variable to change the port Actual binds to.  Then you can map that port to any port you want outside of the container.

See #333 for an example.

**Comment by psybers** [comment]
I think this issue could be closed, as #333 should fix the problem of discoverability on setting custom ports in Docker.

**Comment by hrv231** [comment]
> Can't test this as my docker instalation in armv6 is busted, but from your docker compose file, you're missing port environment variables https://actualbudget.org/docs/config/#port

Correct, by using the ACTUAL_PORT fixed the docker compose issue.

Thanks.
